### PR TITLE
Pocketxtra 17 component buttons

### PR DESCRIPTION
**Pull request by ShubhamBade**
This branch has Primary Secondary and Teriary Button Component. please review it. thank you!!

### CODE REVIEW

File: src/components/buttons/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    responsiveWidth,
+    responsiveFontSize
+} from "react-native-responsive-dimensions";
+import { PrimaryButtonInterface } from '../../interface/buttons/PrimaryButtonInterface';
+import {Colors} from '../../theme/ColorsConstant'
+
+export const PrimaryButton : React.FC<PrimaryButtonInterface & { onPress?: () => void }> = ({
+    title = "",
+    buttonColor = Colors.backgroundColor,
+    height = 8,
```
Comment (tabassum2507) [inline src/components/buttons/PrimaryButton.tsx]: max-height

File: src/components/buttons/PrimaryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    color = Colors.textColor,
+    borderColor=Colors.borderColor,
+    borderWidth=2,
+    borderRadius=22,
+    disabled=true,
+    marginTopLabel=0,
+    marginBottomLabel=0,
+    marginLeftLabel=0,
+    marginRightLabel=0,
+    marginTopButton=0,
```
Comment (tabassum2507) [inline src/components/buttons/PrimaryButton.tsx]: add a space after =

File: src/components/buttons/PrimaryButton.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    marginBottomButton=0,
+    marginLeftButton=0,
+    marginRightButton=0,
+    disabledColor=Colors.disabledTextColor,
```
Comment (tabassum2507) [inline src/components/buttons/PrimaryButton.tsx]: rename it to disabledTextColor

File: src/components/buttons/index.ts [hunk 4]
```diff
@@ -1,4 +1,2 @@
-export { default as ContainedButton } from '../buttons/containedButton';
-
-// export all types of buttons here 
-
+export {PrimaryButton} from "./PrimaryButton"
+export {TertiaryButton} from "./TertiaryButton"
```
Comment (tabassum2507) [inline src/components/buttons/index.ts:2]: add semicolon

File: src/interface/buttons/TertiaryButtonInterface.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    width?: number,
+    fontSize?: number,
+    color?: string,
+    borderColor?: string,
+    borderWidth?: number,
+    borderRadius?: number,
+    icon?: boolean,
+    iconColor?:string,
+    disabledIconColor?:string,
+    sizeIcon?:number,
```
Comment (tabassum2507) [inline src/interface/buttons/TertiaryButtonInterface.ts]: rename to iconSize

**Review by tabassum2507 — APPROVED** [review]

**Review by rohan-arya-w — APPROVED** [review]

**Pull request merged**